### DISCUSSION TITLE
Pass won't open

### DISCUSSION THREAD

**Discussion by NinjaHawkZero** [alexandercerutti/passkit-generator#159] [Q&A] [answered]
Hello!  So I have everything set up with node/express, when I send a request to generate the pass in postman I get a bunch of weird characters which I assume is the image data, and when I download the pass I can't open it.

```js
const stream  = finalPass.getAsStream();

response.set({
	"Content-type": finalPass.mimeType,
	"Content-disposition": `attachment; filename=${passName}.pkpass`,
});

stream.pipe(response)
```

I can show more, cause i'm following pretty closely and can't pinpoint where the problem might be.

**Comment by alexandercerutti** [comment, marked as answer]
Hi @NinjaHawkZero, thanks for using passkit-generator!

Sadly your issue does not contain enough information to let me help you.
You might find helpful the guide I wrote in the wiki on how to debug.

Here it is: https://github.com/alexandercerutti/passkit-generator/wiki/Troubleshooting-(Self-help)#through-console-mac-only

After having debugged the pass, you can come back here and tell me about what you found out, so I can help you! :)

Let me know!

**Reply by NinjaHawkZero** [reply]
You the man!  Learned about console.app, figured it out and fixed it!
